README.md

```diff
@@ -1,11 +1,13 @@
-# CorrelationVector for Go
+# Correlation Vector
 
-[![godoc](https://godoc.org/github.com/Microsoft/CorrelationVector-Go?status.svg)](https://godoc.org/github.com/Microsoft/CorrelationVector-Go)
```

Comment:
Would really prefer if these badges remained.

Reply:
I agree. I retained the badges. Will also include similar badges for other repo's in next version.